src/order-book/api.spec.ts

```diff
@@ -19,7 +19,11 @@ import { BUY_ETH_ADDRESS } from './transformOrder'
 
 enableFetchMocks()
 
-const orderBookApi = new OrderBookApi({ chainId: SupportedChainId.GNOSIS_CHAIN })
+const orderBookApi = new OrderBookApi({ chainId: SupportedChainId.GNOSIS_CHAIN }, undefined, {
+  numOfAttempts: 1,
```

Comment:
Disabled retries for regular API tests (we have `request.test.ts` for testing bacOff)